### PR TITLE
Various improvements, specially related to accesibility

### PR DESCRIPTION
**Pull request by esanzgar**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,64 +1,61 @@
...
-        <li class="home active"><a href="//www.ebi.ac.uk"></a></li>
-        <li class="services"><a href="//www.ebi.ac.uk/services">Services</a></li>
-        <li class="research"><a href="//www.ebi.ac.uk/research">Research</a></li>
-        <li class="training"><a href="//www.ebi.ac.uk/training">Training</a></li>
-        <li class="about"><a href="//www.ebi.ac.uk/about">About us</a></li>
-        <li class="float-right show-for-medium embl-selector">
-          <button class="button float-right" type="button" data-toggle="embl-dropdown">Hinxton</button>
+        <li class="home-mobile" aria-label="Home"><a href="//www.ebi.ac.uk" aria-label="Home"></a></li>
+        <li class="home active" aria-label="Home"><a href="//www.ebi.ac.uk" aria-label="Home"></a></li>
+        <li class="services"><a href="//www.ebi.ac.uk/services" aria-label="Services">Services</a></li>
```
Comment (pwalter-ebi) [inline src/index.html]: Not sure you need an aria-label for a link where it is encapsulating a single word that is the same ???

File: src/index.html [hunk 2]
```diff
@@ -1,64 +1,61 @@
...
+        <li class="research"><a href="//www.ebi.ac.uk/research" aria-label="Research">Research</a></li>
+        <li class="training"><a href="//www.ebi.ac.uk/training" aria-label="Training">Training</a></li>
+        <li class="about"><a href="//www.ebi.ac.uk/about" aria-label="About us">About us</a></li>
+        <li class="float-right show-for-medium embl-selector" aria-label="Hinxton">
+          <button class="button float-right" type="button" data-toggle="embl-dropdown" aria-label="Hinxton">Hinxton</button>
```
Comment (pwalter-ebi) [inline src/index.html]: Double aria-label wrapping same word

File: src/index.html [hunk 3]
```diff
@@ -74,7 +71,7 @@
               <div class="row">
                 <div class="columns medium-3 white-color">
                   <div class="margin-top-xlarge padding-top-xlarge"><!--spacer--></div>
-                  <img src="https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/images/logos/EMBL-EBI/EMBL_EBI_Logo_white.svg" !inline style="width:300px;position:relative;z-index: 5;">
+                  <img alt="EMBL-EBI logo" src="https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/images/logos/EMBL-EBI/EMBL_EBI_Logo_white.svg" style="width:300px;position:relative;z-index: 5;">
```
Comment (pwalter-ebi) [inline src/index.html]: !inline removed in error

**Comment by khawkins98** [comment]
Going to close this one as it will be eclipsed by work on the VF 2.0 migration (most likely).

**Pull request closed without merging**